### DISCUSSION TITLE
Scroll Up to maximize

### DISCUSSION THREAD

**Discussion by LeonardoGentile** [rxhanson/Rectangle#1123] [Ideas]
Hello,
I've been using Hyperdock for years, and one of the features I've been using daily was the possibility to maximize/restore window size by scrolling up/down on the window title.

I'm not sure this is possible right now in Rectangle, if not would you consider adding this feature?

Thanks for your work 👍🙏

**Comment by rxhanson** [comment]
Sorry for the late reply. I think this would fit better in Rectangle Pro, but I'd be fine merging a good PR for it into Rectangle if anyone wants to implement it.
